code/_helpers/global_lists.dm

```diff
@@ -148,6 +149,14 @@ var/global/list/string_slot_flags = list(
 		var/datum/sprite_accessory/marking/M = new path()
 		body_marking_styles_list[M.name] = M
 
+	//Body markings list minus 'non-genetic' markings, indexed by name.
+	paths = typesof(/datum/sprite_accessory/marking/aug)
+	var/list/nonpersist = list()
+	for(var/path in paths)
+		var/datum/sprite_accessory/marking/aug/M = new path()
+		nonpersist[M.name] = M
+	body_marking_nopersist_list = (body_marking_styles_list - nonpersist)
+

```

Comment:
Including a couple lines above this that github won't let me select
```suggestion
	paths = subtypesof(/datum/sprite_accessory/marking)
	for(var/path in paths)
		var/datum/sprite_accessory/marking/M = new path()
		body_marking_styles_list[M.name] = M
		if(!M.genetic)
			body_marking_nopersist_list[M.name] = M
```

And then let
`/datum/sprite_accessory/marking/var/genetic = TRUE` and
`/datum/sprite_accessory/marking/aug/genetic = FALSE`, such that the default behavior is genetic persistence as ON for all existing stuff, and disabled for the augments.

I don't want to start this system as being purely based on types because that might introduce bad assumptions // using a new var allows this to extend more easily to other markings